sorting_test.go

```diff
@@ -257,3 +258,41 @@ func assertRowsEqualByRow[T any](t *testing.T, rowsGot, rowsWant []T) {
 		t.Error(count, "rows mismatched out of", len(rowsWant), "total")
 	}
 }
+
+func TestIssue_82(t *testing.T) {
+	type Record struct {
+		A string `parquet:"a"`
+	}
+
+	fi, err := os.Open("testdata/lz4_raw_compressed_larger.parquet")
```

Comment:
I would add an assertion that this file has more than 1 row group because that's not visible from the test itself.

Reply:
@fpetkovski  That wont be necessary. The number of row groups was not the actual issue. The issue is about mismatch between the rows written/read. 

Go back to the OP ticket you will notice he uses the same test case file as I do there. You can consistently reproduce the issue by running the sample they provided.

Reply:
Got it, thanks for the clarification.